Add links to social media.

AlbertoPdRF opened the pull request:
This is a different proposal to what I said in #3242, so the links have more visibility. Screenshot:

![image](https://user-images.githubusercontent.com/26309166/48285098-c0a80e80-e461-11e8-9404-446d580829d1.png)

Opinions?

Baiqiang commented:
Can you attache a screenshot of mobile?

AlbertoPdRF commented:
Here it is:

![image](https://user-images.githubusercontent.com/26309166/48366012-df4e1580-e6ac-11e8-8afd-63b84b9506b7.png)

It looks bad :confused:

viroulep commented on WcaOnRails/app/views/layouts/_navigation.html.erb:
```diff
@@ -3,6 +3,25 @@
     <div class="navbar-brand">
       <a href="<%= root_path %>"><%= image_tag "wca_logo.svg" %></a>
       <a href="<%= root_path %>"><span>World Cube Association</span></a>
+      |
```
Bootstrap has some [responsive utilities](https://getbootstrap.com/docs/3.3/css/#responsive-utilities), maybe make this a `<span class="hidden-xs hidden-sm">|</span>`, and add a `<span class="visible-xs-block visible-sm-block"></span>` (to add a `display:block` item in the middle of the flow and trigger a "line break" only for small screens)

suushiemaniac commented:
From my personal taste I think it will look better if you use right justification instead of left. So the icons align with the end of "Association" rather than the start of "World".

Jambrose777 commented:
I haven’t found an example of anyone putting social media at the top of a webpage, only the bottom. Are there any examples that others can link?

AlbertoPdRF commented:
@Jambrose777 you can see examples [here](https://www.codeinwp.com/blog/determining-best-placement-social-media-icons/).

operge commented:
I very much like the idea, thanks for the work, Alberto! I was interested in seeing Gregor's idea, so here is my awesome paint skill showing:

![right aligned](https://user-images.githubusercontent.com/6160727/48905236-4a43dd00-ee61-11e8-82d0-43ad8a227fff.jpg)

I am OK either way, but I slightly prefer this to being aligned on the left.

ChrisWrightWCA commented:
I really like the idea and proposed placement though no huge opinion of left/right justification

cubewhiz commented:
I would probably prefer the links in the footer, but currently it takes too long to scroll on the home page to even get to the footer (and generally there is no need to scroll so much), so until we get a new design, they are probably best up top where you have them.

moralsh commented:
I also would prefer them on the footer but look good enough on top for now.. I'm not sure they should be visible on phone,  as they are too small and you might need to zoom in to follow the links.

AlbertoPdRF commented:
With the current home page design, having the icons on the footer is pretty much the same as not having them imo.

Also, not showing the icons on phone would cut drastically the number of people that gets to see (and click on) them. I don't think that zooming in is too much of a problem given that this is something that we have do all day on almost every site anyway.

viroulep commented on WcaOnRails/app/views/layouts/_navigation.html.erb:
```diff
@@ -2,7 +2,28 @@
+      <span>
+        <a href="<%= root_path %>">World Cube Association</a>
+        <br>
+        <a href="https://www.instagram.com/worldcubeassociation/" target="_blank" class="hide-new-window-icon" style="text-decoration:none;">
```
Wow that's three long lines.
I have two requests for these social media links:
  - do not inline style on the tag, use CSS (you can probably add a `a.no-decoration` class somewhere).
  - use rails helpers, it will look much better:
```html
<%= link_to(icon("instagram"), "https://www.instagram.com/worldcubeassociation/", target: "_blank", class: "hide-new-window-icon no-decoration") %>
```

AlbertoPdRF commented:
I think I have addressed your comment, thanks @viroulep! For some reason the `text-decoration:none` doesn't seem necessary anymore, maybe it has to do with the usage of helpers?

viroulep commented:
Merged and deployed!
